src/injection.js

```diff
@@ -1,11 +1,14 @@
-// In order to avoid name conflicts, all global variables used in this injected
-// script are defined as a property of the object `window.nonaltReblog`.
```

Comment:
This comment is for the time when this script was imported as non-module, and it is now obsolete since this script is imported as a module.